src/Features/OnboardingTasks.php

```diff
@@ -165,7 +165,13 @@ public function add_media_scripts() {
 	 * @return array
 	 */
 	public function component_settings( $settings ) {
-		$products = wp_count_posts( 'product' );
```

Comment:
I can't see where the `$products` variable was being used in this method so I removed it while in the file. But cc @joshuatf for 👀

Reply:
Nice- this can definitely be removed.